doc/src/intrinsics-memory.md

```diff
@@ -0,0 +1,92 @@
+# Challenge 2: Verify the memory safery of core intrinsics using raw pointers
+
+- **Status:** Open
+- **Tracking Issue:** *Link to issue*
```

Comment:
Can you please create one and add it here. We can edit the content of the issue later. Thanks!

Reply:
done